geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/KeywordCommand.scala

```diff
@@ -0,0 +1,93 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import com.beust.jcommander.converters.{IParameterSplitter}
+import com.beust.jcommander._
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.tools.accumulo.commands.KeywordCommand.KeywordParameters
+import org.locationtech.geomesa.tools.accumulo.{DataStoreHelper, GeoMesaConnectionParams}
+import org.locationtech.geomesa.tools.common.OptionalFeatureTypeNameParam
+import org.locationtech.geomesa.tools.common.commands._
+
+import scala.collection.JavaConversions._
+import scala.io.StdIn
+
+class KeywordCommand(parent: JCommander) extends Command(parent) {
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType._
+  override val command: String = "keyword"
+  override val params = new KeywordParameters()
+
+  override def execute(): Unit = {
+    val dsParams = new DataStoreHelper(params).paramMap
+    val tryDs = DataStoreFinder.getDataStore(dsParams)
+    if (tryDs == null) {
+      throw new ParameterException("Could not load a data store with the provided parameters")
+    }
+    tryDs.dispose()
```

Comment:
you don't want to call dispose here if you're still using the data store (e.g. on the next line)